src/cdk-experimental/column-resize/column-resize.ts

```diff
@@ -61,6 +63,10 @@ export abstract class ColumnResize implements AfterViewInit, OnDestroy {
     return `cdk-column-resize-${this.selectorId}`;
   }
 
+  setResized() {
+    this.elementRef.nativeElement!.classList.add(WITH_RESIZED_COLUMN_CLASS);
```

Comment:
Why does this need to be called from other classes instead of only responding to the resize stream?

Reply:
I went back and forth on this - I was not sure that applying a min/max size should trigger the notifier, which will eventually have side effects like persisting the size setting.